actionners/calico/networkpolicy/networkpolicy.go

```diff
@@ -18,6 +18,11 @@ import (
 	"github.com/falco-talon/falco-talon/utils"
 )
 
+type Config struct {
+	Allow []string `mapstructure:"allow" validate:"omitempty"`
```

Comment:
FYI, I would like to replace it later this field by 2:
- allow_cidr
- allow_namespaces

Reply:
Ok, with this new structure would be easier to do this validation.